app/configuration/server.py

```diff
@@ -30,7 +32,14 @@ def get_app(self) -> FastAPI:
 
     @staticmethod
     def __register_events(app: FastAPI) -> None:
-        ...
+        app.on_event('startup')(generate_orders)
```

Comment:
Do we really need to run this event on `startup`? Or it's enough to schedule it as a cron event?